db/seedfiles/translations/en/me/insured.rb

```diff
@@ -764,8 +764,8 @@
   :'en.insured.preferences.hbx_registration.bullet2' => "Your right to privacy in deciding whether or not to register to vote",
   :'en.insured.preferences.hbx_registration.bullet3' => "Your right to choose your own political party or other political choice",
   :'en.insured.preferences.hbx_registration.file_a_complaint' => "To file a complaint, contact:",
-  :'en.insured.preferences.voter_registration_description' => 'If you are not registered to vote where you live now and would like to apply to register to vote, visit:',
-  :'en.insured.preferences.voter_registration_link' => 'https://www.maine.gov/sos/cec/elec/voter-info/voterguide.html?ref=voteusa',
+  :'en.insured.preferences.voter_registration_description' => 'If you are not registered to vote where you currently live and would like to register, or need to make updates to your existing registration, go to Maine’s online voter registration service at ',
+  :'en.insured.preferences.voter_registration_link' => 'https://www.maine.gov/vote',
```

Comment:
any idea why this url is being stored in a translation? I get that it is client-specific, but I don't think translation files are the best pattern for that. 

I know this isn't an issue you introduced, but it may be a good idea to fix this while we're here